can-bus/json/FSM/FSM_tx.json

```diff
@@ -1,12 +1,184 @@
 {
-    "FSM_Vitals": {
-        "msg_id": 303,
-        "cycle_time": 1000,
-        "description": "FSM Heartbeat signal",
-        "signals": {
-            "Heartbeat": {
-                "bits": 1
-            }
-        }
-    }
+	"FSM_Errors":{
+		"msg_id": 301,
+		"cycle_time": 1000,
+		"signals": {
+			"PappsOutOfRange": {"bits": 1},
+			"SappsOutOfRange": {"bits": 1},
+			"StackWatermarkAboveThresholdTask1Hz": { "bits": 1 },
+			"StackWatermarkAboveThresholdTask1kHz":{ "bits": 1 },
+			"StackWatermarkAboveThresholdTaskCanTx": { "bits": 1 },
+			"StackWatermarkAboveThresholdTaskCanRx": { "bits": 1 },
+			"WatchdogTimeout": { "bits": 1 },
+			"MissingHeartbeat": {"bits":  1},
+			"WatchdogTimeoutTaskName": { "enum": "RtosTaskName" },
+			"BrakeAccDisagreement": {"enum": "BrakeAccDisagreement"},
+			"SteeringAngleSensorOCSC": {"enum": "SteeringAngleSensorOCSC"},
+			"LeftWheelSpeedOutOfRange": {"enum":  "LeftWheelSpeedOutOfRange"},
+			"RightWheelSpeedOutOfRange": {"enum":  "RightWheelSpeedOutOfRange"},
+			"SteeringAngleOutOfRange": {"enum": "SteeringAngleOutOfRange"},
+			"FrontBrakePressureOutOfRange": {"enum": "FrontBrakePressureOutOfRange"},
+			"RearBrakePressureOutOfRange": {"enum": "RearBrakePressureOutOfRange"},
+			"FlowRateOutOfRange": {"enum": "FlowRateOutOfRange"}
+		}
+	},
+	"FSM_MotorShutdownErrors":{
+		"msg_id": 302,
+		"cycle_time": 1000,
+		"signals": {
+			"PappsOCSCIsActive": {"enum": "PappsOCSC"},
+			"SappsOCSCIsActive": {"enum": "SappsOCSC"},
+			"AppsDisagreementIsActive": {"enum": "AppsDisagreement"},
+			"FlowMeterHasUnderflow": {"enum": "FlowMeterHasUnderFlow"}
+		}
+	},
+	"FSM_CanFifoOverflow":{
+		"msg_id": 303,
+		"cycle_time": 5000,
+		"signals":{
+			"TxOverflowCount": {
+				"bits": 32
+			},
+			"RxOverflowCount": {
+				"bits": 32
+			}
+		}
+	},
```

Comment:
I don't think this needs to be its own message, can you add these signals into FSM_Errors. Also I believe this message is called <board_name>_Warnings on other boards, maybe change it to FSM_Warnings